README.md

```diff
@@ -8,4 +8,8 @@ Some of the images are given here -
 
 "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQKgAjahWEIs10LFxpXY1Q3ihF83q1kd_qNFw&usqp=CAU"
 
-"https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSyv_FXBv-r-NHFVZBuc-WxLAhPjl36SFsVQw&usqp=CAU"
\ No newline at end of file
+"https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSyv_FXBv-r-NHFVZBuc-WxLAhPjl36SFsVQw&usqp=CAU"
+
+"https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSutU6Jmnu7Inir1tI-kr49tcwspmBfJlKEbg&usqp=CAU"
```

Comment:
Consider using a markdown image syntax for displaying images instead of just providing the URLs. This will make the README file more visually appealing and easier to understand. [medium]